src/linter/linter.ts

```diff
@@ -324,17 +328,18 @@ export async function resolveReader({
 
 	let fsBasePath = projectRootDir;
 	let fsBasePathTest = path.join(projectRootDir, "test");
-	let virBasePath = namespace ? `/resources/${namespace}/` : "/resources/";
-	let virBasePathTest = namespace ? `/test-resources/${namespace}/` : "/test-resources/";
+	let virBasePath = "/resources/";
+	let virBasePathTest = "/test-resources/";
 
+	let project;
 	try {
 		const graph = await getProjectGraph(projectRootDir, ui5ConfigPath);
-		const project = graph.getRoot();
+		project = graph.getRoot();
 		projectRootDir = project.getRootPath();
 		fsBasePath = project.getSourcePath();
 		fsBasePathTest = path.join(projectRootDir, project._testPath ?? "test");
 
-		if (!namespace && !project._isSourceNamespaced) {
+		if (!project._isSourceNamespaced) {
```

Comment:
I wonder why this logic here needs to exist. There is already a mapping of the virtual file paths to the filesystem paths in place at https://github.com/SAP/ui5-linter/blob/3694911541e7da5f6581f91fa2504812055056b1/src/linter/linter.ts#L76 which seems to work fine.
I might of course miss some detail here, but I would expect that the very same logic is needed to rewrite the paths so that the fs based ignores / files can be matched.

Reply:
Anyway, you're correct that this functionality is already there in the `lintProject`!
I'll try to figure out if it's possible to refactor it in a way that it also works for `lintFiles`